### ISSUE TITLE
adding folders to docs

### ISSUE THREAD

**Issue by lukehmcc** [riftdweb/rift#36]
Being able to have folders in the docs page would make the experience much better for the user. For example, having a "recipes" folder is just a lot better than having a bunch of dispersed recipes. Though omnisearch does help this a bit, organization is always good. 

Maybe something like this? Keeping it simple is good
![image](https://user-images.githubusercontent.com/58121030/134381758-052bc69e-3683-45a3-a4a5-72b42eee7fb5.png)

**Comment by alexfreska** [comment]
Agreed, folders/nesting documents is coming soon along with omni-search support.
